src/libs/CategoryUtils.ts

```diff
@@ -68,4 +68,14 @@ function getCategoryDefaultTaxRate(expenseRules: ExpenseRule[], categoryName: st
     return categoryDefaultTaxRate;
 }
 
-export {formatDefaultTaxRateText, formatRequireReceiptsOverText, getCategoryApproverRule, getCategoryExpenseRule, getCategoryDefaultTaxRate};
+function updateCategoryInMccGroups(mccGroups: never[] | Record<string, MccGroup>, oldCategoryName: string, newCategoryName: string): Record<string, MccGroup> {
+    const updatedGroups: Record<string, MccGroup> = {};
+
+    for (const [key, group] of Object.entries(mccGroups || {})) {
+        updatedGroups[key] = group.category === oldCategoryName ? {...group, category: newCategoryName, pendingAction: 'update'} : group;
```

Comment:
I'm getting the following error on this line. Could you take a look at this please?

<img width="758" alt="image" src="https://github.com/user-attachments/assets/59970508-f177-4da9-8f8f-a7e8c9e20fa8">

Reply:
@eVoloshchak Could you reload your IDE? I asked because I didn’t encounter any errors on my side. The pipeline ESLint also passed. I think your IDE hasn’t initialized TypeScript yet.
<details>
<summary>Video</summary>
https://github.com/user-attachments/assets/431849c7-57cd-4806-b93c-1fcc35d10f37

</details>

Reply:
Reloading the IDE didn't fix it for me, the problem still persists. 
Looks like the `never` type is the problem, we shouldn't use it. I'll post the draft changes before, it does resolve the error for me, but please double check on your end